mmeds/database/metadata_uploader.py

```diff
@@ -204,16 +195,28 @@ def run(self):
             self.user_id = 1
         self.check_file = fig.DATABASE_DIR / 'last_check.dat'
 
-        # Create a copy of the Data file
-        datafile_copies = {key: create_local_copy(Path(filepath).read_bytes(),
-                                                  filepath, self.path.parent)
-                           for key, filepath in self.datafiles.items()
-                           if filepath is not None}
-        self.import_metadata(**datafile_copies)
-
+        # Create symbolic links to data file directories
+        self.mongo_import()
+        """
```

Comment:
the docstring is missing its lower half so this function appears commented out

Reply:
yeah that's not a docstring, it's supposed to be commented out. gonna remove